### ISSUE TITLE
How does the LL affect model covergence?

### ISSUE THREAD

**Issue by weberxie** [NVIDIA/nccl#369]
Hi team, I'm training the MobileNetV2 with ImageNet on 4 V100 GPUs, the environment is:

- NCCL 2.4.7 
- PyTorch 1.2
- Horovod 0.17

there are 2 variables for 4 experiments, each experiment run 5 epochs, the result below,

NCCL_LL_THRESHOLD is 1048576 and 4 GPU on 4 nodes:
TRAIN Iter 12540: lr = 0.499960,	loss = 4.210017,	Top-1 err = 0.728484,	Top-5 err = 0.496907,	data_time = 0.001105,	train_time = 0.170369

NCCL_LL_THRESHOLD is 1048576 and 4 GPU on 1 node:
TRAIN Iter 12540: lr = 0.499960,	loss = 4.059321,	Top-1 err = 0.725821,	Top-5 err = 0.492524,	data_time = 0.003998,	train_time = 0.191447

NCCL_LL_THRESHOLD is 0 and 4 GPU on 4 nodes:
TRAIN Iter 12540: lr = 0.499960,	loss = 4.079141,	Top-1 err = 0.723485,	Top-5 err = 0.490941,	data_time = 0.000144,	train_time = 0.173883

NCCL_LL_THRESHOLD is 0 and 4 GPU on 1 node:
TRAIN Iter 12540: lr = 0.499960,	loss = 3.996922,	Top-1 err = 0.730930,	Top-5 err = 0.499446,	data_time = 0.001104,	train_time = 0.207908

Conclusion: for Top-1 err, if we set NCCL_LL_THRESHOLD to 0, it will be much better, but if we set NCCL_LL_THRESHOLD to 0, the experiment run on 4 nodes is much better than run on 1 node.

So, anyone can answer my doubts on this issue? Thanks in advance.

**Comment by stephenmsachs** [comment]
### Issue Cleanup: Helping Us Focus on Current Challenges

  We're reviewing older issues to ensure we prioritize the most relevant and active ones. Since this issue hasn't seen updates in over 6 months, we'll be closing it for now.

  ,*This change helps us focus our efforts on addressing any current issues our users are facing.* If this issue still affects you, please don't hesitate to reopen it with a quick update (e.g., "Still relevant on [version=X]").
  Thanks for your understanding and for contributing to NCCL.
